swap/src/monero/wallet_rpc.rs

```diff
@@ -146,12 +148,18 @@ impl WalletRpc {
 
         let mut reader = BufReader::new(stdout).lines();
 
-        while let Some(line) = reader.next_line().await? {
-            if line.contains("Starting wallet RPC server") {
-                break;
-            }
+        // If we do not hear from the monero_wallet_rpc process for 3 seconds we assume
+        // it is is ready
+        while let Ok(res) = tokio::time::timeout(Duration::from_secs(3), reader.next_line()).await {
```

Comment:
I know that technically, `res` is not the read line but the `Result` but can we still call it `line`? Makes it easier to understand I think.

Reply:
fixed